packages/react-microlink/src/components/Card/CardMedia/image.js

```diff
@@ -1,6 +1,8 @@
 import MediaWrap from './wrap'
 
-export default MediaWrap.extend`
+export default MediaWrap.extend.attrs({
```

Comment:
wut, what is `extend.attrs`? 🤔

Reply:
It's to pass attributes directly to the styled-component, without the need for an "extra wrap". Explained in their [docs](https://www.styled-components.com/docs/basics#attaching-additional-props).